Cargo.lock

```diff
@@ -0,0 +1,32 @@
+# This file is automatically @generated by Cargo.
```

Comment:
Why was the lockfile committed?

Reply:
I added a few dev-dependencies that have an MSRV that doesn't work on the stable used for this branch. Since I only listed 3 dependencies in the lock-file, the rest should still automatically pick the newest version.

Reply:
I guess this means that checkout of this branch in an environment where the lockfile already exists will cause conflicts.

Since `predicates` is a dev-dependency, I think using "=" requirement would be fine.

Reply:
That would work too. I'm going to keep this because I already spent a bunch of time figuring out all of the needed versions with this approach.

Reply:
(Well, this is a branch for an old version that only a few contributors use, so I think either one is fine.)